src/models/network.ts

```diff
@@ -39,8 +40,9 @@ const mainnet = Networks.add({
   pubkeyhash: versionBytes['mainnet']['p2pkh'],
```

Comment:
This comment is in the wrong line. I thought we would rename it to `htr-mainnet` and also we would remove all non-Hathor networks from bitcore. Why aren't we doing that?

Reply:
We cannot remove non-Hathor networks from bitcore since the `remove` method of bitcore is not working properly.

Reply:
I think the idea is to run the remove method even if it's not working properly. It's better for us to get an empty network and raise an error instead of getting a bitcoin network.

Reply:
This wouldn't work, many places in bitcore use `Network.get(arg) || Network.defaultNetwork` which returns the livenet (even if we remove it). Later on the code it will search for this network and it will not find it generating an error (usually `"version argument is not a buffer, it's undefined"`). It broke 31 out of 137 of our tests when i tried.